lib/usd/translators/jointWriter.cpp

```diff
@@ -136,7 +137,6 @@ static GfMatrix4d _GetJointWorldBindTransform(const MDagPath& dagPath)
         CHECK_MSTATUS_AND_RETURN(status, GfMatrix4d(1));
         MPlug plgWorldMatrices = fnNode.findPlug("worldMatrix", false, &status);
         CHECK_MSTATUS_AND_RETURN(status, GfMatrix4d(1));
-        TF_VERIFY(membersIdx < plgWorldMatrices.numElements());
```

Comment:
Can you comment on why this check wasn't good?

Reply:
It's because `membersIdx` is a logical index, while `numElements()` gives the number of physical plugs... so the two aren't related.